### PR TITLE
Proposed edits for Ch.1

### PR DESCRIPTION
**Pull request by wonal**
Hello SerHack!
I wrote to you on Taiga last week about proofreading.  
Most of the changes I've made are on the grammatical side or just shifting words/sentences around to help the flow/how it sounds.  I'm sure there are better editors in the Monero community, so just consider this an opinion, and I hope this is helpful. 
Here is a summary of the edits I've made/am proposing:

The first section seemed more on the topic of blockchains, so I changed the header to reflect that.  I'm unsure about the opening example;  I think a different example would be better-suited, although I had trouble thinking of one.  If you are trying to show the reader that the current state of the financial system carries risks of trusting a centralized party, I'm not sure if a friend lending money to another friend helps set the stage for that.    

I tended to make your writing more formal -- removing some direct references to the reader ("you"), and also trying to make the writing a little less biased :) so removing words like "most powerful".  However, making it more personal to the reader could have been completely intentional, so ignore those edits if that's how you wanted them to be!

### CODE REVIEW

**Comment by serhack** [comment]
Hello, yesterday I forgot to rewrite the new content ("Why open source" section), could you please send me a feedback about that part?

I really like what you wrote, but I have a doubt about example: what example should I use? the main problem is "trusting" "the digital system" that could be edited.

**Comment by 4matter** [comment]
Great edits/feedback. I agree that a stronger example can be used at the beginning. Though, it would be nice to keep it an ELI5-style example. 

Serhack, maybe add a stronger example as an "Issue"?

**Comment by serhack** [comment]
@4matter #2
